assets/js/util.js

```diff
@@ -293,13 +300,82 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    apyRequestStartTime = Date.now();
     if(requestUrl.length > THRESHOLD_REQUEST_URL_LENGTH) {
         requestOptions.type = 'POST';
         return $.ajax(requestOptions);
     }
     return $.jsonp(requestOptions);
 }
 
+function checkServiceLoadTimes(requestDuration) {
+    var lastNRequestsDuration = [];
+    var storedLastNRequestsDuration = [];
+    var demoThreshold = 10000; // for demo purposes only
+    try {
+        window.sessionStorage;
```

Comment:
I thought about this a bit more and I don't think these need to be stored in `sessionStorage`. 

From our conversation on the PR:

> My current understanding is, for the current session ie until user closes the tab/window, I store all the page load times in a session variable and then calculate the average?

This is fine. Simple and easy.

From this I assumed that session storage persisted only through the life of the tab but it seems to be the life of the browser instead. I think it makes more sense (and drastically simplifies the code) for this to just be per page. We should also make sure to only display the message to the user once. We don't want them to keep seeing it.